README.md

```diff
@@ -22,15 +22,28 @@ The following are needed for building/running KWASM:
 On Ubuntu >= 15.04 (for example):
 
 ```sh
-sudo apt-get install make gcc maven openjdk-8-jdk flex opam pkg-config libmpfr-dev autoconf libtool pandoc zlib1g-dev
+sudo apt install                                                         \
+        autoconf curl flex gcc libffi-dev libmpfr-dev libtool make maven \
+        opam openjdk-8-jdk pandoc pkg-config python3 python-pygments     \
```

Comment:
I'm very skeptical of this list of dependencies also, because it includes some things from the other list that surely are either extraneous or refer only to the jello paper. I really feel strongly that this needs to be better tested or we're going to give users a bad install experience

Reply:
I would prefer to keep the Dockerfile's having the same prefix, since it will mostly be used on our CI servers.